README.md

```diff
@@ -10,32 +10,90 @@ In this workshop, we're going to:
 [wasmtime]: https://wasmtime.dev
 [bca]: https://bytecodealliance.org
 
-### Prerequisites
+## 📦 1. Setup
+
 > [!NOTE]
 > This first example is Rust-focused, but we'll move onto working with other languages (Go, TypeScript, Python) in a later tutorial stage.
 
 > [!NOTE]
 > If you prefer to not clone and install things locally, feel free to work from a Docker container with:
 >
 > `docker run --rm -it rust:1-slim-buster`.
+>
+> Once you're in the Docker container, you can install the basic dependencies for the demo with:
+>
+> `apt update; apt install curl pkg-config -y;`
```

Comment:
Request now that I found out more things:
```suggestion
> `apt update; apt install curl pkg-config libssl-dev git -y;`
```

Reply:
Done!